roles/glance/tasks/main.yml

```diff
@@ -43,6 +43,12 @@
     - etype: other
       permission: r
 
+- name: install bbc glance plugins
+  pip:
+    name: "git+git://github.com/blueboxgroup/bbc-openstack-plugins.git#egg=bbc_openstack_plugins"
```

Comment:
In addition to this there are other changes needs to glance/horizon config for url method to work.